src/Check.hs

```diff
@@ -90,13 +91,30 @@ checkTestsBuildReport False =
 checkTestsBuildReport True =
   "- The tests defined in `passthru.tests`, if any, passed"
 
+versionWithoutPath :: String -> Text -> String
+versionWithoutPath resultPath expectedVersion = do
```

Comment:
`do` (syntactic sugar to insert the `Monad` operators `>>` or `>>=`) is useless here, you're returning a `String` not a `Monad m => m String`